modules/nw-sriov-configure-exclude-topology-manager.adoc

```diff
@@ -0,0 +1,214 @@
++
+[source,terminal]
+----
+$ oc create -f sriov-policy-numa.yaml
+----
++
+.Example output
+[source,terminal]
+----
+sriovnetworknodepolicy.sriovnetwork.openshift.io/policy-for-numa-0 created
+----
+
+. Create the `SriovNetwork` CR:
+
+.. Save the following YAML in the `sriov-network-numa.yaml` file, replacing values in the YAML to match your environment:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-numa-0-network <1>
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: sriovnuma0 <2>
+  networkNamespace: <namespace> <3>
+  ipam: |- <4>
+    {
+      "type": "<ipam_type>", 
+    }
+----
+<1> Replace `sriov-numa-0-network` with the name for the SR-IOV network resource. 
+<2> Specify the resource name for the `SriovNetworkNodePolicy` CR from the previous step. This YAML uses a sample `resourceName` value. 
+<3> Enter the namespace for your SR-IOV network resource.
+<4> Enter the IP address management configuration for the SR-IOV network.
+
+.. Create the `SriovNetwork` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-network-numa.yaml
+----
++
+.Example output
+[source,terminal]
+----
+sriovnetwork.sriovnetwork.openshift.io/sriov-numa-0-network created
+----
+
+. Create a pod and assign the SR-IOV network resource from the previous step:
+
+.. Save the following YAML in the `sriov-numa-pod.yaml` file, replacing values in the YAML to match your environment:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: <pod_name>
+  annotations:
+    k8s.v1.cni.cncf.io/networks: |-
+      [
+        {
+          "name": "sriov-numa-0-network", <1>
+        }
+      ]
+spec:
+  containers:
+  - name: <container_name>
+    image: <image> 
+    imagePullPolicy: IfNotPresent
+    command: ["sleep", "infinity"]
+----
+<1> This is the name of the `SriovNetwork` resource that uses the `SriovNetworkNodePolicy` resource.
+
+.. Create the `Pod` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-numa-pod.yaml
+----
++
+.Example output
+[source,terminal]
+----
+pod/example-pod created
+----
+
+.Verification
+
+. Verify the status of the pod by running the following command, replacing `<pod_name>` with the name of the pod:
++
+[source,terminal]
+----
+$ oc get pod <pod_name>
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE
+test-deployment-sriov-76cbbf4756-k9v72   1/1     Running   0          45h
+----
+
+. Open a debug session with the target pod to verify that the SR-IOV network resources deployed to a different node than the memory and CPU resources.
+
+.. Open a debug session with the pod by running the follow command, replacing <pod_name> with the target pod name. 
++
+[source,terminal]
+----
+$ oc debug pod/<pod_name>
+----
++
+[source,terminal]
+----
+$ chroot /host
+----
```

Comment:
s $ chroot /host to # chroot /host

When I invoke this command in a procedure, I break it into a separate step. Not sure it this is "required."

```
. Set `/host` as the root directory within the debug shell. The debug pod mounts the root file system from the host in `/host` within the pod. By changing the root directory to `/host`, you can run binaries from the host file system:
+
[source,terminal]
----
# chroot /host
----
```